packaging/_parser.py

```diff
@@ -0,0 +1,155 @@
+from typing import Any, List, Tuple
+
+from ._tokenizer import Tokenizer
+
+
+def parse_named_requirement(requirement: str) -> Tuple[Any, str, Any, str, str]:
+    """
+    NAMED_REQUIREMENT: NAME EXTRAS* URL_SPEC (SEMICOLON + MARKER)*
+    """
+    tokens = Tokenizer(requirement)
+    name = tokens.read("IDENTIFIER").text.strip("'\"")
```

Comment:
What does the `strip("'\"")` part do? I don't think the NAME part can contain quotes. Should this be `strip()`?

Reply:
You are right, the `strip()` is unnecessary and can be removed entirely.